src/masternodes/mn_checks.cpp

```diff
@@ -1547,11 +1547,7 @@ class CCustomTxApplyVisitor : public CCustomTxVisitor
         CDataStructureV0 liveKey{AttributeTypes::Live, ParamIDs::Economy, EconomyKeys::DFIP2203Current};
         auto balances = attributes->GetValue(liveKey, CBalances{});
 
-        // Can be removed after the hard fork, since it will be backward compatible
-        // but have to keep it around for pre 2.8.0 nodes for now
-        if (height >= static_cast<uint32_t>(consensus.FortCanningCrunchHeight)) {
-            CalculateOwnerRewards(obj.owner);
-        }
+        CalculateOwnerRewards(obj.owner);
```

Comment:
This needs to remain guarded. Otherwise full sync nodes will end up with some accounts that performed a FutureSwap before FCC having a higher balance than others which could lead to consensus issues.

Reply:
But each action that does require a balance in consensus (expect for this bug), should already be doing is expected to do this action anyway?

Is there anything else I'm missing - otherwise, let's just do a confirmation anyway with full a sync and proceed to remove?